lib/rspec/rails/matchers/have_enqueued_mail.rb

```diff
@@ -89,6 +89,10 @@ def arguments_match?(job)
           super(job)
         end
 
+        def verify_arguments_match_signature!(_job, _args)
+          # Not implemented for mailers
+        end
```

Comment:
This no-op in is a temporary placeholder because `HaveEnqueuedMail` inherits from `ActiveJob::HaveEnqueuedJob`.

If this PR is welcome I also have a branch which adds a similar verification of arguments to mailers:
https://github.com/odlp/rspec-rails/compare/pr-verify-job-perform-signature...odlp:rspec-rails:pr-verify-mailer-signature

Happy to raise a PR for this too

Reply:
I'm happy for you to include that in this PR 👍

Reply:
Great, updated 👍